packages/device/src/classes/device.ts

```diff
@@ -1,8 +1,7 @@
 import { Channel, Socket } from 'phoenix';
 import { Player, Playlist, Renderer, Viewport, Layer } from '@castmill/player';
 import { ResourceManager, Cache, StorageIntegration } from '@castmill/cache';
-
-import { Machine } from '../interfaces/machine';
+import { Machine, type DeviceInfo } from '../interfaces/machine';
```

Comment:
I was not familiar with this syntax. Does it just import the types? (although in this case as they are interfaces it should be the same or?

Reply:
It just imports the types and is completely removed in the compiled code and is sometimes needed to get rid of circular references. However, the general opinion seems to be that typescript should handle this automatically and that this syntax should only be used where needed. Will remove...

Reply:
changed type import to normal